File: domaene/src/test/java/com/github/haschi/haushaltsbuch/domaene/RunCukesTest.java
package com.github.haschi.haushaltsbuch.domaene;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@CucumberOptions(
    monochrome = true,
    strict = false,
    plugin = {"pretty", "html:target/site/cucumber-pretty", "json:target/cucumber.json"},
    tags = "~@ignore")
@RunWith(Cucumber.class)
public class RunCukesTest {
}

File: testit/src/test/kotlin/com/github/haschi/domain/haushaltsbuch/RunCukesTest.kt
package com.github.haschi.domain.haushaltsbuch

import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber
import org.junit.runner.RunWith

@CucumberOptions(
        strict = true,
        tags = ["@domain"],
        plugin = ["progress"])
@RunWith(Cucumber::class)
class RunCukesTest

